token/client/src/token.rs

```diff
@@ -894,6 +916,88 @@ where
         .await
     }
 
+    /// Wrap lamports into native account
+    pub async fn wrap<S: Signers>(
```

Comment:
i liked the idea of making this immutable by default and exposing a second "you asked for it, you got it" function, in favor of taking a bool or making mutable owner the default, but id be open to either of those interfaces if you prefer one